### PR TITLE
Ks/finalize frontpage v2

### PR DESCRIPTION
**Pull request by ks10825n**
This will render our user lists for each user
Need to connect request to the server to update user's book list

### CODE REVIEW

File: client/src/components/LeftComponent/BooksRead/BooksRead.jsx [hunk 1]
```diff
@@ -71,7 +72,7 @@ const BooksRead = ({ completed, removeFromCompleted, completedToBookClub }) => {
                     subheader={
                       <ThemeProvider theme={theme}>
                         <Typography gutterBottom variant='subtitle2'>
-                          {datum.author}
+                          {datum.authors[0]}
```
Comment (anerolabs) [inline client/src/components/LeftComponent/BooksRead/BooksRead.jsx:75]: Can you change to display all authors?

**Review by anerolabs — CHANGES_REQUESTED** [review]
Looks good- just please update the book lit items to render all authors in the userBook data. Thanks

File: client/src/components/LeftComponent/BooksRead/BooksRead.jsx (hunk 1 above)
Comment (ks10825n) [reply client/src/components/LeftComponent/BooksRead/BooksRead.jsx]: Added!

**Review by anerolabs — APPROVED** [review]
Looks good!

**Pull request merged**